scss/_variables.scss

```diff
@@ -563,7 +563,8 @@ $container-padding-x: $grid-gutter-width !default;
 $border-width:                .125rem !default;
 $border-widths: (
   1: $border-width * .5,
-  2: $border-width
+  2: $border-width,
+  3: $border-width * 1.5
```

Comment:
By doing this you're adding a `.border-3` utility so it must be documented in https://deploy-preview-1242--boosted.netlify.app/docs/5.1/utilities/borders/#width by adding the following example: `<span class="border border-3"></span>`.

Note: an impact is observed where `.border-3` is used in the docs but is handle directly via https://github.com/twbs/bootstrap/pull/36476.

Reply:
Done